recipe/meta.yaml

```
@@ -22,7 +21,7 @@ requirements:
     - wheel
   run:
     - python
-    - typing_extensions # [py<38]
+    - typing_extensions >=4
```

```suggestion
    - typing_extensions >=4 # [py<311]
```
https://github.com/django/asgiref/blob/e38d3c327c01aa82c0bf2726220700c1097ea6cc/setup.cfg#L35